src/shaders/line.fragment.glsl

```diff
@@ -24,6 +25,23 @@ varying vec2 v_tex_b;
 uniform sampler2D u_gradient_image;
 #endif
 
+#ifdef DEPTH_OCCLUSION
+uniform sampler2D u_depth;
+uniform float u_depth_occlusion_factor;
+
+varying vec4 v_projected_pos;
+
+float depth_occlusion_visibility(vec4 frag) {
+    vec3 coord = frag.xyz / frag.w;
+    vec2 uv = 0.5 * coord.xy + 0.5;
+    float buffer_depth = unpack_depth(texture2D(u_depth, uv));
+    float occlusion = step(buffer_depth, coord.z);
+    return 1.0 - occlusion * u_depth_occlusion_factor;
+}
+#else
+float depth_occlusion_visibility(vec4 frag) { return 1.0; }
```

Comment:
Final nit: is the `else` clause required? :)

Reply:
Removed